packages/@ionic/lab/stencil.config.js

```diff
@@ -8,6 +8,7 @@ exports.config = {
   ],
   copy: [
     { src: 'assets' },
+    { src: '../../../../../node_modules/@ionic-internal/ionic-ds/www/assets/fonts', dest: 'assets/fonts' },
```

Comment:
You can use `require.resolve('@ionic-internal/ionic-ds')` to get the path and it will work no matter where it's installed